PaolaSantolloV/react-apprenticeship-capstone4
# Feat/deliverable4

## Description
This PR contain the 4 deliverable: React Router, Global State Management, Design Patterns, Best Practices & Performance Optimizations

- Add checkout page
- Add cart page
- Add functionality of cart
- Add "add to cart" functionality on botton


## Type of Change



## How should this be tested?
In your terminal run` npm install` after that ` npm run start`, now you can see the page on[ localhost:3000](https://localhost:3000/)
To run the test `npm run test`, if you want to watch the coverage `npm run test a -- --coverage`.

## What is the deliverable?
deliverable 4 

## Screenshots and links
netlify link: [https://quizzical-mcclintock-3f0c04.netlify.app/](https://quizzical-mcclintock-3f0c04.netlify.app/)


## Checklist
